services/blockchain-indexer/shared/database/schema/blockchainApps.js

```diff
@@ -0,0 +1,33 @@
+/*
+ * LiskHQ/lisk-service
+ * Copyright © 2022 Lisk Foundation
+ *
+ * See the LICENSE file at the top-level directory of this distribution
+ * for licensing information.
+ *
+ * Unless otherwise agreed in a custom licensing agreement with the Lisk Foundation,
+ * no part of this software, including this file, may be copied, modified,
+ * propagated, or distributed except according to the terms contained in the
+ * LICENSE file.
+ *
+ * Removal or modification of this copyright notice is prohibited.
+ *
+ */
+module.exports = {
+	primaryKey: 'chainID',
+	schema: {
+		chainID: { type: 'string' },
+		name: { type: 'string' },
+		state: { type: 'string' },
+		address: { type: 'string' },
+		lastUpdated: { type: 'string' },
+		lastCertificateHeight: { type: 'string' },
+	},
+	indexes: {
+		chainID: { type: 'key' },
+		state: { type: 'key' },
+		address: { type: 'key' },
+		lastCertificateHeight: { type: 'key' },
```

Comment:
We do not require indexes on `address` and `lastCertificateHeight` rather we need it on `name` to support the `search` param.
```suggestion
		name: { type: 'key' },
```